doc/nrf-bm/release_notes/release_notes_0.8.0.rst

```diff
@@ -0,0 +1,257 @@
+
+IDE and tool support
+********************
+
+`nRF Connect extension for Visual Studio Code <nRF Connect for Visual Studio Code_>`_ is the recommended IDE for |BMshort| v0.8.0.
+See the :ref:`install_nrf_bm` section for more information about supported operating systems and toolchain.
+
+Supported boards
+****************
+
+* PCA10156 (`nRF54L15 DK`_)
+
+Migration from nRF5 SDK
+***********************
+
+For guidelines on how to migrate an application built on nRF5 SDK to |BMshort|, refer to `Migration notes - nRF5 SDK to nRF Connect SDK Bare Metal`_.
+
+Changelog
+*********
+
+The following sections provide detailed lists of changes by component.
+
+SDK installation
+================
+
+* The default way to install |BMshort| is through the |nRFVSC| user interface, using either the :guilabel:`Pre-packaged SDK & Toolchain` option or a GitHub installation.
+  For detailed steps, see :ref:`cloning_the_repositories_nrf_bm`.
+
+S115 SoftDevice
+===============
+
+* Updated the S115 SoftDevice version to ``s115_9.0.0-3.prototype``.
+  The SoftDevice comes in three variants to support different SoCs of the nRF54L Series: nRF54L15, nRF54L10, and nRF54L05.
+
+  * :file:`s115_9.0.0-3.prototype_nrf54l15_softdevice.hex`
+  * :file:`s115_9.0.0-3.prototype_nrf54l10_softdevice.hex`
+  * :file:`s115_9.0.0-3.prototype_nrf54l05_softdevice.hex`
+
+  For more details about this release of the SoftDevice, see :ref:`s115_docs`.
+
+  For API documentation, see `S115 9.0.0-3.prototype API reference`_.
+
+SoftDevice Handler
+==================
+
+* Added support for seeding the SoftDevice RNG when requested by the SoftDevice.
+
+* Fixed the maximum MTU Kconfig option.
+
+Boards
+======
+
+* Added the following ``bm_nrf54l15dk`` board variants for MCUboot (DFU-enabled board variants):
+
+  * ``bm_nrf54l15dk/nrf54l05/s115_softdevice/mcuboot``
+  * ``bm_nrf54l15dk/nrf54l10/s115_softdevice/mcuboot``
+  * ``bm_nrf54l15dk/nrf54l15/s115_softdevice/mcuboot``
+
+..
+
+* Renamed the ``bm_nrf54l15dk`` board variants:
+
+  * ``bm_nrf54l15dk/nrf54l05/softdevice_s115`` to ``bm_nrf54l15dk/nrf54l05/s115_softdevice``
+  * ``bm_nrf54l15dk/nrf54l10/softdevice_s115`` to ``bm_nrf54l15dk/nrf54l10/s115_softdevice``
+  * ``bm_nrf54l15dk/nrf54l15/softdevice_s115`` to ``bm_nrf54l15dk/nrf54l15/s115_softdevice``
+
+..
+
+* Fixed:
+
+  * Fixed wrong RRAM size on the nRF54L10 emulation target from 1022 to 1012 KiB.
+
+..
+
+* Removed:
+
+  * The template board variants.
+  * The ``no_softdevice`` variants of the ``bm_nRF54l15dk``.
+
+DFU
+===
+
+* DFU based on MCUboot Loader mode is now introduced in the SDK:
+
+  * Experimental support added for DFU (update of Firmware Loader, SoftDevice, and application).
+    This is an asymmetric foreground DFU bank solution - the user application bank and SofDevice are much bigger than the update Loader.
+  * This solution contains a port of the MCUmgr SMP protocol part required for performing the DFU operation.
```

Comment:
a port? There is no port, it includes it as is, just say it uses MCUmgr to load firmware updates using image management.

Reply:
this point is now removed as per previous comments